server/context.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
+
+const PageFeatures = createContext(null);
```

Comment:
Note for future reference that you put the context in the server folder - it is a client thing only so it doesn't belong there. There will be probably a folder named `contexts` for those

Reply:
not relevant anymore, but noted